### PR TITLE
Issue 2 511

### PR DESCRIPTION
**Pull request by FocusZhouGD**
Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Review by dengliming — COMMENTED** [review]
I think `label ` column is too long.

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 1]
```diff
@@ -20,233 +20,234 @@ USE `soul`;
...
-  `rpc_ext` varchar(512) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL DEFAULT NULL COMMENT 'rpc的扩展信息，json格式',
-  `date_created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT '创建时间',
-  `date_updated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT '更新时间',
-  `enabled` tinyint(4) NOT NULL DEFAULT 0 COMMENT '启用状态',
+  `app_name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'application name',
+  `path` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'path, cannot be repeated',
+  `path_desc` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'path description',
+  `rpc_type` varchar(64) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'rpc type',
+  `service_name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL DEFAULT NULL COMMENT 'service name',
+  `method_name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL DEFAULT NULL COMMENT 'Method name',
```
Comment (yiwenlong) [inline soul-admin/src/main/resources/META-INF/schema.sql]: 我觉得应该保持统一的句首大小写书写习惯。

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 2]
```diff
@@ -20,233 +20,234 @@ USE `soul`;
...
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('25', '12' ,'minimumNumberOfCalls','Error minimum calculation threshold', 1, 2, 2, '2020-11-28 11:28:34', '2020-11-28 11:28:34');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('26', '12' ,'waitIntervalFunctionInOpenState','Fuse opening duration', 1, 2, 2, '2020-11-28 11:29:01', '2020-11-28 11:29:01');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('27', '12' ,'permittedNumberOfCallsInHalfOpenState','Half open threshold', 1, 2, 2, '2020-11-28 11:29:55', '2020-11-28 11:29:55');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('28', '12' ,'failureRateThreshold','fuse error rate', 1, 2, 2, '2020-11-28 11:30:40', '2020-11-28 11:30:40');
```
Comment (yiwenlong) [inline soul-admin/src/main/resources/META-INF/schema.sql]: Values 第二列后面的 , 不应该放在空格后面。

Comment (yiwenlong) [reply soul-admin/src/main/resources/META-INF/schema.sql]: 除非必要，修改应该保持最小改动原则。

**Review by tydhot — APPROVED** [review]

**Review by tydhot — CHANGES_REQUESTED** [review]
I think we should unify the case of words

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 3]
```diff
@@ -181,73 +181,83 @@ CREATE TABLE IF NOT EXISTS `soul_dict` (
...
 INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `date_updated`) VALUES ('8','springCloud','1', '0', ' 2018-06-25 13:47:57', '2018-06-25 13:47:57');
 INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `date_updated`) VALUES ('9','hystrix', '0','0', ' 2020-01-15 10:19:10', '2020-01-15 10:19:10');
 INSERT IGNORE INTO `plugin` (`id`, `name`,`role`, `enabled`, `date_created`, `date_updated`) VALUES ('10','sentinel', '1','0', ' 2020-11-09 01:19:10', '2020-11-09 01:19:10');
-INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `config`, `enabled`, `date_created`, `date_updated`) VALUES ('11','sofa', '0', ' {"protocol":"zookeeper","register":"127.0.0.1:2181"}', '0', '2020-11-09 01:19:10', '2020-11-09 01:19:10');
+INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `config`, `enabled`, `date_created`, `date_updated`) VALUES ('11','sofa', '0', ' {"protocol":"zookeeper","register":"127.0.0.1:2181"}', '0', '2020-11-09 01:19:10', '2020-11-09 01:19: 10');
```
Comment (tydhot) [inline soul-admin/src/main/resources/META-INF/schema.sql:208]: '2020-11-09 01:19: 10' the blank here will cause exception when the someone query here

**Review by tydhot — CHANGES_REQUESTED** [review]

File: soul-admin/src/main/resources/META-INF/schema.sql (hunk 3 above)
Comment (FocusZhouGD) [reply soul-admin/src/main/resources/META-INF/schema.sql:208]: ok，i will check

Comment (FocusZhouGD) [reply soul-admin/src/main/resources/META-INF/schema.sql:208]: ok，i will check thinks

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 4]
```diff
@@ -181,73 +181,83 @@ CREATE TABLE IF NOT EXISTS `soul_dict` (
...
 
 /*plugin*/
 INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `date_updated`) VALUES ('1','sign','1', '0', ' 2018-06-14 10:17:35', '2018-06-14 10:17:35');
 INSERT IGNORE INTO `plugin` (`id`, `name`,`role`,`config`,`enabled`, `date_created`, `date_updated`) VALUES ('2','waf', '1',' {"model":"black"}','0', '2018-06-23 10:26:30', '2018-06-13 15:43:10');
 INSERT IGNORE INTO `plugin` (`id`, `name`,`role`, `enabled`, `date_created`, `date_updated`) VALUES ('3','rewrite', '1','0', ' 2018-06-23 10:26:34', '2018-06-25 13:59:31');
-INSERT IGNORE INTO `plugin` (`id`, `name`,`role`,`config`,`enabled`, `date_created`, `date_updated`) VALUES ('4','rate_limiter','1',' {"master":"mymaster","mode":"Standalone","url":"192.168.1.1:6379","password":"abc"}', '0', '2018-06-23 10:26:37', '2018-06-13 15:34:48');
+INSERT IGNORE INTO `plugin` (`id`, `name`,`role`,`config`,`enabled`, `date_created`, `date_updated`) VALUES ('4','rate_limiter','1',' {"master":"mymaster","mode":"standalone","url":"192.168.1.1:6379","password":"abc"}', '0', '2018-06-23 10 :26:37', '2018-06-13 15:34:48');
```
Comment (songyuequan) [inline soul-admin/src/main/resources/META-INF/schema.sql:201]: '2018-06-23 10 :26:37'  the blank here will cause exception when the someone query here

**Review by songyuequan — CHANGES_REQUESTED** [review]

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 5]
```diff
@@ -181,73 +181,83 @@ CREATE TABLE IF NOT EXISTS `soul_dict` (
...
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('25', '12' ,'minimumNumberOfCalls','enabledrror minimum calculation threshold', 1, 2, 2, '2020-11-28 11:28:34', '2020-11-28 11:28:34');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('26', '12' ,'waitIntervalFunctionInOpenState','degrade opening duration', 1, 2, 2, '2020-11-28 11:29:01', '2020-11-28 11:29:01');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('27', '12' ,'permittedNumberOfCallsInHalfOpenState','half open threshold', 1, 2, 2, '2020-11-28 11:29:55', '2020-11-28 11:29:55');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('28', '12' ,'failureRateThreshold','degrade ailure rate', 1, 2, 2, '2020-11-28 11:30:40', '2020-11-28 11:30:40');
+
```
Comment (songyuequan) [inline soul-admin/src/main/resources/META-INF/schema.sql:254]: too many empty lines

**Review by songyuequan — APPROVED** [review]

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 6]
```diff
@@ -212,42 +212,42 @@ INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `d
...
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('25', '12' ,'minimumNumberOfCalls','enabledrror minimum calculation threshold', 1, 2, 2, '2020-11-28 11:28:34', '2020-11-28 11:28:34');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('26', '12' ,'waitIntervalFunctionInOpenState','degrade opening duration', 1, 2, 2, '2020-11-28 11:29:01', '2020-11-28 11:29:01');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('27', '12' ,'permittedNumberOfCallsInHalfOpenState','half open threshold', 1, 2, 2, '2020-11-28 11:29:55', '2020-11-28 11:29:55');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('28', '12' ,'failureRateThreshold','degrade ailure rate', 1, 2, 2, '2020-11-28 11:30:40', '2020-11-28 11:30:40');
```
Comment (tydhot) [inline soul-admin/src/main/resources/META-INF/schema.sql:253]: Miss f?

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 7]
```diff
@@ -212,42 +212,42 @@ INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `d
...
-INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('26', '12' ,'waitIntervalFunctionInOpenState','Fuse opening duration', 1, 2, 2, '2020-11-28 11:29:01', '2020-11-28 11:29:01');
-INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('27', '12' ,'permittedNumberOfCallsInHalfOpenState','Half open threshold', 1, 2, 2, '2020-11-28 11:29:55', '2020-11-28 11:29:55');
-INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('28', '12' ,'failureRateThreshold','fuse error rate', 1, 2, 2, '2020-11-28 11:30:40', '2020-11-28 11:30:40');
+INSERT IGNORE INTO plugin_handle (`id`,`plugin_id`,`field`,`label`,`data_type`,`type`,`sort`,`date_created`,`date_updated`) VALUES ('25', '12' ,'minimumNumberOfCalls','enabledrror minimum calculation threshold', 1, 2, 2, '2020-11-28 11:28:34', '2020-11-28 11:28:34');
```
Comment (tydhot) [inline soul-admin/src/main/resources/META-INF/schema.sql:250]: enabled error ?

**Review by tydhot — CHANGES_REQUESTED** [review]

**Review by tydhot — APPROVED** [review]

**Pull request merged**